doc/develop/coding-guidelines.rst

```diff
@@ -212,3 +212,51 @@ For e.g. 4.1 standard and internal imports
                                 MD_AT_TEMPSPECTRUM, MD_AT_FIB)
 
 
+Naming Convention for Pull Requests, Branches and Commits
+==========================================================
+
+* The pull request and branch name should be the same.
+
+    * Recommended naming convention: *[ticket] title*.
+    * It is recommended to use the *[]* only for the pull request name.
+|
+* A pull request should contain a minimal description of the changes and what problem they solve.
+
+    * One can include images, links, and tables to help convey this information.
+    * Example template of a pull request description:
+
+        .. code-block::
+
+			## Describe your changes
+
+			## Ticket number and link
+|
+* A commit message must contain a title and body.
+
+    * Recommended naming convention:
+    * for the commit title: *[label] ~ 80 characters (in imperative mood, lowercase)*. Find the Note below.
+    * for the commit body: *try to explain what and why, not how (motivation)*.
+    * Please do leave a blank line between the title and body.
```

Comment:
Can you add here or in the PR description part that if a pull request contains only a single commit it is recommended to use the commit message as pull request description.